v-next/hardhat-node-test-reporter/integration-tests/index.ts

```diff
@@ -1,24 +1,22 @@
+import { readFileSync, readdirSync, statSync } from "node:fs";
```

Comment:
can you explain why we're using the sync versions?

Reply:
This doesn't run anything in parallel, so there's no value in using async versions, and it was just complicating a `filter()` call. If this were a test run by the actual test runner, I'd use async versions, but it's a top-level single-purpose script.

Reply:
I understand the call, but I'm uneasy about starting to use `node:fs` directly. If we add exceptions, this practice might spread, and we could miss out on good error handling and other checks. Do you think it's worth adding the sync versions to the utils?

Reply:
Oh, I see. Your question is not about `sync`/`async`, but I guess you want to know why not the utils package. 

That's not possible because it would introduce a circular dependency between projects, and the typescript will complain. 

The reporter used to use the utils package, but as soon as I started using the reporter in the utils package, that error appeared.

I think it's better to have a nicer reporter in -utils than to avoid importing fs directly here.

Reply:
Got it. Makes sense under that scenario 👍